generators/dockertools/index.js

```diff
@@ -288,12 +299,12 @@ module.exports = class extends Generator {
 			dockerFileTools: 'Dockerfile-tools',
 			imageNameRun: `${applicationName.toLowerCase()}-flask-run`,
 			imageNameTools: `${applicationName.toLowerCase()}-flask-tools`,
-			buildCmdRun: 'python -m compileall server test',
-			testCmd: 'python -m unittest tests.app_tests.ServerTestCase',
-			buildCmdDebug: 'python -m compileall server test',
+			buildCmdRun: 'python -m compileall .',
```

Comment:
Good idea but we need to add a flag under [Starter-Bluemix-Basic](https://github.ibm.com/arf/starter-bluemix-basic) to show that we are getting a project from someplace other than a generated starter. In addition, we need to add some a flag under [generator-readme](https://github.ibm.com/arf/generator-readme) to document to the user if he uses `bx dev enable` you will need to manually modify the `cli-config` to fit their usages.

Reply:
Okay, I changed back `testCmd` and `debugCmd`, but what are your thoughts on just keeping the `buildCmd`s as `python -m compileall .` instead of `python -m compileall server test`? If the user ever adds a new directory for additional python files, the `compileall` will still hit them without concern. Performance wise, this shouldn't really make much of a difference I'd imagine.

Reply:
Yeah this is fine.